### PR TITLE
card styles

### PR DESCRIPTION
**Pull request by stormcloud266**
Adds the styles and the remove passkey flow to the passkey list 

Screenshot 2024-05-16 at 2 03 51 PM
Screenshot 2024-05-16 at 2 04 05 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/419?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#420** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/420?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#419** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/419?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#418** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/418?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/components/icons/IconDots.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <Svg
+      viewBox="0 0 4 16"
+      color={color as ColorTokens | undefined}
+      width={size ?? 4}
+      height={size ?? 16}
+      fill={'none'}
+      {...rest}
+    >
+      <Path
+        fill="#40FB50"
```
Comment (0xBigBoss) [inline packages/app/components/icons/IconDots.tsx:18]: nbd, pretty sure having the fill like this kills the color attr above.

File: packages/app/features/account/settings/backup/index.tsx [hunk 2]
```diff
@@ -126,55 +144,195 @@ const SendAccountCredentials = ({
...
-      </Paragraph>
-      <H5 fontWeight={'300'} color={'$color05'}>
-        Key Slot
-      </H5>
-      <Paragraph fontWeight={'300'} color={'$color05'} fontFamily={'$mono'}>
-        {cred.key_slot}
-      </Paragraph>
+    <YStack
+      w={'100%'}
+      gap={'$size.1.5'}
```
Comment (0xBigBoss) [inline packages/app/features/account/settings/backup/index.tsx:149]: these sizes always weird me out lol. like how does this even work. good stuff.

File: packages/app/features/account/settings/backup/index.tsx [hunk 3]
```diff
@@ -126,55 +144,195 @@ const SendAccountCredentials = ({
...
+                    case activeSigningKeysError !== null:
+                      return (
+                        <Paragraph
+                          maxWidth={'600'}
+                          fontFamily={'$mono'}
+                          fontSize={'$5'}
+                          color={'$color12'}
+                        >
+                          {activeSigningKeysError?.message ??
+                            `Something went wrong: ${activeSigningKeysError}`}
```
Comment (0xBigBoss) [inline packages/app/features/account/settings/backup/index.tsx:225]: sry i forgot to add this. but we should simplify the error message. split the error message on `.` and take the first element. prevents getting crazy looooonnnnngggg RPC errrors.

```suggestion
                            `Something went wrong: ${activeSigningKeysError?.message.split('.').at(0) ?? `Something went wrong`}
```

File: packages/app/features/account/settings/backup/index.tsx [hunk 4]
```diff
@@ -126,55 +144,195 @@ const SendAccountCredentials = ({
...
-                  Passkey is not confirmed onchain. Finish confirming the passkey onchain.
-                </Paragraph>
-                <Button theme="warning" {...link}>
-                  Confirm
-                </Button>
-              </>
+              <RemovePasskeyConfirmation
+                name={webauthnCred.display_name}
+                onCancel={() => setCardStatus('default')}
+                onRemove={() => setCardStatus('default')}
```
Comment (0xBigBoss) [inline packages/app/features/account/settings/backup/index.tsx:200]: throw an error so we remember to implement. unless ur planning on implementing in the stack.

File: packages/app/utils/formatTimeDate.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+export function formatTimeDate(dateStr: string) {
+  const date = new Date(dateStr)
+
+  if (Number.isNaN(date.getTime())) {
+    return 'Invalid date'
+  }
+
+  const formattedTime = date
+    .toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true })
```
Comment (0xBigBoss) [inline packages/app/utils/formatTimeDate.ts:9]: this is a good idea, but don't force `en-US` on users. use undefined instead to use their selected locale.

```suggestion
    .toLocaleTimeString(undefined, { hour: 'numeric', minute: 'numeric', hour12: true })
```

Comment (0xBigBoss) [reply packages/app/utils/formatTimeDate.ts:9]: same for below.

File: packages/app/components/icons/IconDots.tsx (hunk 1 above)
Comment (stormcloud266) [reply packages/app/components/icons/IconDots.tsx:18]: Yesss good catch!

**Review by 0xBigBoss — APPROVED** [review]

**Comment by stormcloud266** [comment]
### Merge activity

* **May 17, 5:19 PM EDT**: @stormcloud266 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/419).

**Pull request merged**